NEWS.md

```diff
@@ -2,9 +2,13 @@
 
 ## General
 
-* Added support for models of class `hglm` (*hglm*), `mblogit` (*mclogit*) and
+* Added support for models of class `hglm` (*hglm*), `mblogit` (*mclogit*), and
   `fixest_multi` (*fixest*).
 
+* `as.data.frame` methods for regression model objects have been removed. The
+  `model_parameters()` function is the preferred solution to tidy regression
+  model objects into a data frame.
+  
```

Comment:
I think their intention was not to be an alias for `model_parameters()`, but rather to behave like `as.data.frame.brmsfit()` and return posterior draws via `bootstrap_model()`.

Reply:
But where is this used?

Reply:
Or the change you are requesting is only editing the NEWS item?

Reply:
> I don't think that the as.data.frame() methods for bootstrapping should be retired. They provide useful parity between Stan models and other models

I agree they're convenient, but the problem here is that we're introducing a breaking change into the behaviour of `as.data.frame()` for base-R objects. This _can_ lead to unintended behaviour and non-working code that is hard to debug then. We had this with `datawizard::data_read()`, which checks whether an object can be coerced to data frame, and if not, returns the object itself (which is sometimes intended, when you have an RDS file with a model object, e.g.). The tests worked well and everything passed, but then one test was run before, which loaded the _parameters_ package. The RDS file, containing an `lm` object, now suddenly returned a data frame for `data_read()`, which led to weird results and we first couldn't figure out why.

Since the `as.data.frame()` methods really just call `bootstrap_model()`, there's not much lost to retire them.

Reply:
I think the convenience afforded by these methods is more than offset by the potential problems they can cause, and that too in an unpredictable and mysterious ways, as we painfully discovered in the case of _datawizard_.